src/platforms/hydrabus/README.md

```diff
@@ -23,16 +23,18 @@ Connections
 How to Build
 ------------
 
-```sh
+``` sh
 cd blackmagic
-make clean
-make PROBE_HOST=hydrabus
+meson setup build --cross-file=cross-file/hydrabus.ini
+cd build
+meson compile
 ```
 
 How to Flash the firmware with Windows
 --------------------------------------
 
-* After build:
-  * 1) Download files from https://github.com/hydrabus/hydrafw/tree/master/utils/windows_dfu_util
-  * 2) Force the F4 into system bootloader mode by jumpering "BOOT0" to "3V3" and "PB2/BOOT1" to "GND" and reset (RESET button). System bootloader should appear.
-  * 3) Run the command `DfuSeCommand.exe -c --de 0 -d --fn .\src\blackmagic.dfu`
+After build:
+  1. Download files from https://github.com/hydrabus/hydrafw/tree/master/utils/windows_dfu_util
+  2. Run the command: `python3 ../scripts/dfu-convert.py -i blackmagic_hydrabus_firmware.bin blackmagic_hydrabus_firmware.dfu`
```

Comment:
Ah. This script has been removed as we weren't aware of it's need/use here. Perhaps that should be fixed in a follow-up PR.